packer/install-dependencies.sh

```diff
@@ -48,16 +48,14 @@ cat << EOF >> ${RELEASE_NOTES_FILEPATH}
 EOF
 
 if [[ ${FEATURE_FLAGS} == *"docker-engine"* ]]; then
-    DOCKER_ENGINE_REPO="https://apt.dockerproject.org/repo"
-    installDockerEngine
-    overrideDockerEngineStorageDriver
-    echo "  - docker-engine v${DOCKER_ENGINE_VERSION}" >> ${RELEASE_NOTES_FILEPATH}
-    installGPUDrivers
-    echo "  - nvidia-docker2 nvidia-container-runtime" >> ${RELEASE_NOTES_FILEPATH}
+    # deprecated
+    exit 1
 else
     MOBY_VERSION="3.0.4"
     installMoby
     echo "  - moby v${MOBY_VERSION}" >> ${RELEASE_NOTES_FILEPATH}
+    installGPUDrivers
```

Comment:
Since we are now installing this on the "regular" VHD flavor, we should remove it at provisioning for VMs that aren't GPU enabled.